x/staking/module.go

```diff
@@ -3,14 +3,47 @@ package staking
 import (
 	"encoding/json"
 
+	"github.com/cosmos/cosmos-sdk/codec"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/cosmos/cosmos-sdk/x/auth"
 	"github.com/cosmos/cosmos-sdk/x/staking/types"
 	abci "github.com/tendermint/tendermint/abci/types"
 )
 
+// app module basics object
+type AppModuleBasic struct{}
```

Comment:
uhh this file doesn't have any colors 🤷‍♂️ ?

Reply:
what do you mean? like your language syntax colour utility is working here? works for me and on github!